.github/workflows/update-version.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Update veda-ui version
       shell: bash
       run: |
-        yarn install @teamimpact/veda-ui@${{ github.event.client_payload.VERSION_NUMBER }}
+        yarn add @teamimpact/veda-ui@${{ github.event.client_payload.VERSION_NUMBER }}
```

Comment:
would you rather do explicit update command? https://classic.yarnpkg.com/lang/en/docs/cli/upgrade/